ttnn/cpp/ttnn/operations/eltwise/unary/common/unary_op_utils.cpp

```diff
@@ -340,6 +340,12 @@ std::pair<string, string> get_op_init_and_func_default(UnaryOpType op_type, std:
         case UnaryOpType::IDENTITY_UINT32:
             op_init_and_name = {"identity_tile_init();", fmt::format("identity_tile_uint32({});", idst)};
             break;
+        case UnaryOpType::FLOOR:
+            op_init_and_name = {"floor_tile_init();", fmt::format("floor_tile({});", idst)};
+            break;
+        case UnaryOpType::FLOOR_FLOAT32:
+            op_init_and_name = {"floor_tile_init();", fmt::format("floor_tile_float32({});", idst)}; break;
+            break;
```

Comment:
It seems the only purpose of adding a new UnaryOpType that is unique to FLOAT32 is because we want a slightly different set of defines  in std::pair<std::string, std::string> op_init_and_name;

The decision for the op type was based on the dtype() at runtime.    For now I don't see a way to get around this with the current architecture.  I'd like us to really think through how we can do this better though.